vite.config.js

```diff
@@ -178,10 +178,11 @@ module.exports = ({ command, mode }) => {
       },
       outDir: path.resolve(__dirname, 'heroku/static'),
     },
+    worker: {
+      format: 'es'
```

Comment:
I don't recall this before, it is because the dynamic import?

Reply:
Yes, I'll add a comment